package.json

```diff
@@ -4,25 +4,17 @@
   "description": "LN donations page",
   "main": "index.js",
   "scripts": {
+    "lint": "eslint src/*.js",
     "start": "cross-env NODE_ENV=development webpack-dev-server --open",
     "build": "cross-env NODE_ENV=production webpack",
-    "format": "prettier --write 'packages/**/*.js'",
-    "precommit": "lint-staged"
-  },
-  "lint-staged": {
-    "*.{js,jsx,css,md}": [
-      "prettier --write --single-quote",
-      "eslint --fix",
-      "git add"
-    ]
+    "format": "prettier --write 'packages/**/*.js'"
   },
   "repository": "https://github.com/lncm/donations",
   "author": "meeDamian",
   "license": "MIT",
   "private": false,
   "engines": {
-    "node": ">=11",
-    "npm": ">=6"
+    "node": ">=10"
```

Comment:
Shouldn't that be sth like `=10`?  Doesn't it go for the newer major, if available?

Reply:
Will use `v10` + `minor` + `patch`